zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+                hasPermission(session, PUBLISHER_PERMISSIONS);
+    }
+
+    /**
+     * @param email the email of the user to check.
+     * @return boolean
+     * @throws IOException
+     */
+    @Override
+    public boolean isPublisher(String email) throws IOException {
+        boolean result = true;
+        if (StringUtils.isEmpty(email)) {
+            warn().user(email)
+                    .log("request for isPublisher unsuccessful empty email");
+            result = false;
+        } else {
+            try {
+                Session s = getSessionfromEmail(email);
+                if (!hasPermission(s, PUBLISHER_PERMISSIONS)) {
+                    warn().user(email)
+                            .log("request for isPublisher privilages check unsuccessful");
+                    result = false;
+                }
+
+            } catch (Exception exception) {
+                warn().user(email)
+                        .log("request for publisher privilages exception caught");
+                result = false;
+            }
+        }
+        return result;
+    }
+
+    /**
+     * @param session {@link Session} to get the user details from.
+     * @return boolean if valid session with email and has admin permissions
+     * @throws JWTVerificationException
+     */
+    @Override
+    public boolean isAdministrator(Session session) throws JWTVerificationException {
+        return session != null && !StringUtils.isEmpty(session.getEmail()) &&
+                hasPermission(session, ADMIN_PERMISSIONS);
+    }
+
+    /**
+     * @param email the email of the user to check.
+     * @return
+     * @throws JWTVerificationException
+     */
+    public boolean isAdministrator(String email) throws JWTVerificationException {
+        boolean result = true;
+        if (StringUtils.isEmpty(email)) {
+            warn().user(email)
+                    .log("request for isAdministrator unsuccessful empty email");
+            result = false;
+        } else {
+            try {
+                Session s = getSessionfromEmail(email);
+                if (!hasPermission(s, ADMIN_PERMISSIONS)) {
+                    warn().user(email)
+                            .log("request for admin privilages check unsuccessful");
+                    result = false;
+                }
+
+            } catch (Exception exception) {
+                warn().user(email)
+                        .log("request for admin privilages exception caught");
+                result = false;
+            }
+        }
+
+        return result;
+    }
+
+
+    /**
+     * @param collection the collection to check users against.
+     * @return
+     * @throws JWTVerificationException
+     */
+
+    @Override
+    public List<User> getCollectionAccessMapping(Collection collection) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
```

Comment:
Can we specifically state in the meth Javadoc that  this functionality is not supported by this implementation and it will throw an exception if called.

Can we do this for every method that is not supported please - its important to document it for future users to know it was intentional.